lab_1_classify_by_unigrams/main.py

```diff
@@ -64,12 +85,21 @@ def calculate_mse(predicted: list, actual: list) -> float | None:
 
     In case of corrupt input arguments, None is returned
     """
+    if len(predicted) != len(actual):
+        return None
+    if not (all(isinstance(p, (float, int)) for p in predicted) and all(isinstance(a, (float, int))
+                                                                        for a in actual)):
+        return None
+    mse = 0
+    quantity = len(actual)
+    for i in range(quantity):
+        mse += (actual[i] - predicted[i]) ** 2
+    return round((mse / len(actual)), 4)
```

Comment:
Are you sure you need the internal brackets?

Reply:
as for the line `return round((mse / len(actual)), 4)`, yes, because otherwise it gives an error